### PR TITLE
Resources Health Check ReadMe

### PR DESCRIPTION
**Pull request by Tratcher**
null

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/extensions/pull/4678)

### CODE REVIEW

**Review by RussKie — APPROVED** [review]

File: src/Libraries/Microsoft.Extensions.Diagnostics.HealthChecks.ResourceUtilization/README.md [hunk 1]
```diff
@@ -18,6 +18,50 @@ Or directly in the C# project file:
...
+public static IHealthChecksBuilder AddResourceUtilizationHealthCheck(this IHealthChecksBuilder builder, Action<ResourceUtilizationHealthCheckOptions> configure, IEnumerable<string> tags)
+```
+
+For example:
+
+```csharp
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddHealthChecks()
+    .AddResourceUtilizationHealthCheck(o =>
```
Comment (danmoseley) [inline src/Libraries/Microsoft.Extensions.Diagnostics.HealthChecks.ResourceUtilization/README.md:42]: just curious, is 'o' just the convention in extensions land for the param going to these builder delegates? or it is meant to stand for something

Comment (Tratcher) [reply src/Libraries/Microsoft.Extensions.Diagnostics.HealthChecks.ResourceUtilization/README.md:42]: It is shorthand for `options`.

File: src/Libraries/Microsoft.Extensions.Diagnostics.HealthChecks.ResourceUtilization/README.md [hunk 2]
```diff
@@ -18,6 +18,50 @@ Or directly in the C# project file:
...
+    });
+
+var app = builder.Build();
+
+app.MapHealthChecks("/healthz");
+
+app.Run();
+```
+
+`CpuThresholds` and `MemoryThresholds`'s percentages default to `null` and will not be report as degraded or unhealthy unless configured. The `SamplingWindow` defaults to 5 seconds.
```
Comment (sebastienros) [inline src/Libraries/Microsoft.Extensions.Diagnostics.HealthChecks.ResourceUtilization/README.md:64]:
```suggestion
`CpuThresholds` and `MemoryThresholds`'s percentages default to `null` and will not be reported as degraded or unhealthy unless configured. The `SamplingWindow` defaults to 5 seconds.
```

**Review by joperezr — APPROVED** [review]
LGTM. Let me know when ready to merge.

**Pull request merged**